shared/api/endpoints/submitMalteHelpForm.ts

```diff
@@ -37,15 +37,15 @@ const contactChannelText = (contactChannel: ContactChannel): string => {
 }
 
 const buildHelpFormBody = ({
-  languageCode,
+  url,
   name,
   roomNumber,
   contactChannel,
   additionalContactInformation,
   contactGender,
   comment,
 }: BuildHelpFormBodyParams): string => `
-Sprache: ${languageCode}
+Gesendet von ${url}
```

Comment:
Why no more language? Because its part of the url?

Reply:
Jep exactly. Do you think I should readd it? I think just the BCP tag feels a little bit weird here. The url provides more context.

Reply:
No, I guess its fine. But probably more of a question for the maltesers/@osmers.

Reply:
Hm, I think that might be fine - but for non-technical users maybe a bit more difficult to read? Maybe we can even use the language name instead of the code?

Having the URL definitely helps though bcs we also need to be able to send out emails only to certain groups based on the region the form was submitted in.

Reply:
Jep but we only have the "native" language names in the app, right? Not sure if they could understand what فارسی means 🤔